README.md

```diff
@@ -2,120 +2,87 @@
+Join our open discussions and share news:
 
 - [Email list](https://lists.cncf.io/g/cncf-tag-security)
 - [CNCF Slack](https://slack.cncf.io/) #tag-security channel
 
-Leadership:
-
-- To reach the leadership team (chairs & tech leads), email
-  [cncf-tag-security-leads@lists.cncf.io](mailto:cncf-tag-security-leads@lists.cncf.io)
-- To reach the chairs, email [cncf-tag-security-chairs@lists.cncf.io](mailto:cncf-tag-security-chairs@lists.cncf.io)
-
-### Slack governance
-
-Refer to the [slack governance document](slack.md) for details on slack channels
-and posting to the channels.
-
-## Meeting times
-
-For our members in North and South America, we host weekly sessions each Wednesday at 10 am (UTC-7). To participate, simply use the following Zoom link: <https://zoom.us/j/99809474566>. The meeting ID is 998 0947 4566.
+## Meeting Information
 
-Meanwhile, participants from Europe, the Middle East, and Africa (EMEA) can join bi-weekly meetings on Wednesdays at 1 pm UTC+0, which adjusts to UTC+1 when daylight saving time is in effect. Join us through this Zoom link: <https://zoom.us/j/99917523142>, with the meeting ID: 999 1752 3142.
+- **Americas**: Weekly on Wednesdays at 10 am (UTC-7). [Zoom link](https://zoom.us/j/99809474566), Meeting ID: 998 0947 4566.
+- **EMEA**: Bi-weekly on Wednesdays at 1 pm UTC+0 (adjusts for daylight saving). [Zoom link](https://zoom.us/j/99917523142), Meeting ID: 999 1752 3142.
 
-To find the corresponding time in your local area, please see your timezone [here](https://time.is/).
+Check your local timezone [here](https://time.is/). Meetings are listed on the [CNCF calendar](https://www.cncf.io/calendar/) and the [TAG Security Calendar](https://calendar.google.com/calendar/u/0?cid=MGI4dTVlbDh0YTRzOTN0MmNtNzJ0dXZoaGtAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ).
 
-This dual schedule ensures that no matter where you are, you'll have a place in our conversations.
-
-We invite you to mark your calendars and join the dialogue. For your convenience, all meetings are listed on the main [CNCF calendar](https://www.cncf.io/calendar/) as well as the [TAG Security Calendar](https://calendar.google.com/calendar/u/0?cid=MGI4dTVlbDh0YTRzOTN0MmNtNzJ0dXZoaGtAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ). These calendars are updated regularly to ensure that you stay informed of all upcoming meetings and events.
-
-Got something to bring up or share? Review how to get a topic or presentation added to the Agenda on our [process](governance/process.md#getting-on-the-agenda) page.
+To add a topic to the agenda, review our [process](governance/process.md#getting-on-the-agenda).
 
 ## Gatherings
 
-Please let us know if you are going and if you are interested in attending (or
-helping to organize!) a gathering. Create
-a [github issue](https://github.com/cncf/tag-security/issues/new) for an event
-and add to list below:
+Interested in attending or organizing an event? Create a [GitHub issue](https://github.com/cncf/tag-security/issues/new):
```

Comment:
Why an issue for attending an event?

Reply:
Cargo cult. Will remove.

Reply:
After further consideration, I've noticed that people typically don't initiate events by filing an issue. Instead, they organize and plan events independently through platforms like Luma or Meetup, or through KCD. Therefore, I removed the suggestion to file a GitHub issue. If anyone is organizing an event, they can handle it themselves and announce it on social media, but it won't be considered our official event. I have retained SecurityCon and past events in the list.